cfpb/sbl-frontend
# [e2e] Make tests runnable against dev deployment

Made tests configurable through environment variables in preparation for running against the dev deployment

## Changes

- Created environment variables for test
- Added Constants file to pull in the environment variables for test
- Updated hard coded locations to use the values in the Constants file instead
- Updated playwright config to pull in the base url as an environment variable

## How to test this PR

1. Read the Notes
2. Pull the branch
3. Relaunch the stack if necessary
4. Run the e2e tests and verify that they all still pass
5. Play with the envars if you wish and see that the behavior changes as the envars do

## Notest

- There appears to be a test failure in InstitutionProfile.spec.ts that is coming from Design System React. 
  - If you encounter this, you can ignore it.
